### PR TITLE
Rename RCTWKWebView to CRAWKWebView to avoid symbol clash with RN 0.57

### LINKED ISSUE #194: Incompatible with React Native 0.57
In 57 WkWebView support has been added and they using exact same file names and classes RCTWKWebView.m(h) so it won't compile now.

Seems like we should rename files and classes in this repo to be able to compile with React Native 0.57.

### PR DESCRIPTION
**Pull request by headlessme**
Fixes #194

### CODE REVIEW

**Comment by kirillplatonov** [comment]
We really need this in master since it's not possible to upgrade to RN 0.57. And RN < 0.57 doesn't work with Xcode 10

**Comment by oferRounds** [comment]
+

File: ios/RCTWKWebView/RCTWKWebView.m [hunk 1]
```diff
@@ -611,8 +611,13 @@ - (void)webViewWebContentProcessDidTerminate:(WKWebView *)webView
...
 - (void)webView:(WKWebView *)webView decidePolicyForNavigationResponse:(WKNavigationResponse *)navigationResponse decisionHandler:(void (^)(WKNavigationResponsePolicy))decisionHandler {
   if (_onNavigationResponse) {
-    NSDictionary *headers = ((NSHTTPURLResponse *)navigationResponse.response).allHeaderFields;
-    NSInteger statusCode = ((NSHTTPURLResponse *)navigationResponse.response).statusCode;
+    NSDictionary *headers = @{};
+    NSInteger statusCode = 200;
+    if([navigationResponse.response isKindOfClass:[NSHTTPURLResponse class]]){
+        headers = ((NSHTTPURLResponse *)navigationResponse.response).allHeaderFields;
+        statusCode = ((NSHTTPURLResponse *)navigationResponse.response).statusCode;
+    }
```
Comment (TheAlmightyBob) [inline ios/RCTWKWebView/RCTWKWebView.m]: How is this part related to the symbol clash and rename?

Comment (pdfun) [reply ios/RCTWKWebView/RCTWKWebView.m]: RN 0.57 duplicate class `RCTWebView` ` RCTWebViewManager`. Error will occur when Xcode build.
image

Comment (TheAlmightyBob) [reply ios/RCTWKWebView/RCTWKWebView.m]: I know about the duplicate symbol error. But the specific lines of code I commented on do not relate to those symbols. It looks like a different commit accidentally came along for the ride.

Comment (insraq) [reply ios/RCTWKWebView/RCTWKWebView.m]: I think this happens to be a fix for #199

Comment (insraq) [reply ios/RCTWKWebView/RCTWKWebView.m]: I think this happens to be a fix for #199

**Comment by insraq** [comment]
I am on vacation now and I will likely merge it after I am back

**Comment by insraq** [comment]
on npm now: 2.0.0

I have also renamed files and updated the example project to RN 0.57.
